Automattic/simplenote-android
# Update Gradle Plugin

### Fix
Update the Gradle wrapper and plugin to the most recent versions.  The `variant.generateBuildConfig` and `tasks.copyGradlePropertiesExampleFile.execute()` tasks were updated as well since they are no longer supported.

### Test
- Verify continuous integration build succeeds.
- Verify app builds and runs without error.

### Review
@aforcier and @maxme, I requested your review since you have experience with these files.  @bummytime, I requested you as a reviewer to keep you informed of Simplenote changes.  Only one developer is required to review these changes, but anyone can perform the review.
